packages/documentation/src/sidebar.js

```diff
@@ -16,47 +16,42 @@ module.exports = {
               name: '2. Access internal tools',
               href: '/getting-started/internal-tools',
             },
-            {
-              name: '3. Leverage CI workflow',
-              href: '/getting-started/workflow',
-            },
-          ],
-        },
-        {
-          name: 'Project and environment structure',
-          items: [
-            {
-              name: 'VA.gov environments',
-              href: '/getting-started/environments',
-            },
-            {
-              name: 'Bundles and Code organization',
-              href: '/getting-started/bundles-and-code-organization',
-            },
           ],
         },
         {
           name: 'Common tasks',
           items: [
             {
-              name: 'Running VA.gov locally',
-              href: '/getting-started/common-tasks/local',
-            },
-            {
-              name: 'Building VA.gov',
-              href: '/getting-started/common-tasks/build',
+              name: 'Run and Build VA.gov',
+              href: '/getting-started/common-tasks/run-build',
             },
             {
-              name: 'Running tests',
-              href: '/getting-started/common-tasks/tests',
+              name: 'Run tests',
+              href: '/getting-started/common-tasks/test',
             },
             {
-              name: 'Linting your code',
-              href: '/getting-started/common-tasks/lint',
+              name: 'VA.gov workflow',
```

Comment:
I think we should move this up a level and make it it's own section in between the start up guide and common tasks. It doesn't feel like a task to me, but I like the organization of the information here.

Reply:
![Screen Shot 2019-04-19 at 12 10 00](https://user-images.githubusercontent.com/4998130/56437311-1350e680-629c-11e9-89a4-1a87fbd4a068.png)